docs/source/configuration.rst

```diff
@@ -88,6 +88,37 @@ in projects which utilise a lot of complicated templating.
 
 .. _templateconfig:
 
+Rule Configuration
+------------------
+
+Rules can be configured with the `.sqlfluff` config files.
+
+Common rule configurations can be set in the *[sqlfluff:rules]* section.
+
+For example:
+
+.. code-block:: cfg
+
+    [sqlfluff:rules]
+    unquoted_identifiers_policy = all
+    max_line_length = 88
```

Comment:
I think it would be worth including all the common ones here to make it more obvious which ones go in this section:

```suggestion
    # Some rules can be configured directly from the config common to other rules.
   [sqlfluff:rules]
   tab_space_size = 4
   max_line_length = 80
   indent_unit = space
   comma_style = trailing
   allow_scalar = True
   single_table_references = consistent
   unquoted_identifiers_policy = all
```

Reply:
I committed the suggestion, but in what sense are those "all the common ones"?

Reply:
I got them from the default config: https://github.com/sqlfluff/sqlfluff/blob/main/src/sqlfluff/core/default_config.cfg

As everything needs to be added to that AFAIK I think that is them all

Reply:
Unfortunately that means that it can be easy to miss adding it here if something new is added. But I suppose we don't promise here that the list would be comprehensive. It's just an example.

Reply:
Yeah we’ll try to keep it in sync but may will miss something. Still I think it’s good to start with what we know now.